package.json

```diff
@@ -5,8 +5,8 @@
     "test": "./node_modules/.bin/elm-verify-examples --elm-test=elm-test"
   },
   "dependencies": {
-    "elm": "^0.19.0-bugfix2",
-    "elm-test": "^0.19.0-beta11",
-    "elm-verify-examples": "^3.0.1"
+    "elm": "^0.19.0-bugfix6",
```

Comment:
Bump to latest versions (this has the cascading effects in the package-lock.json)